staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -249,14 +268,32 @@ func (o *AuditWebhookOptions) AddFlags(fs *pflag.FlagSet) {
 		"Strategy for sending audit events. Blocking indicates sending events should block"+
 			" server responses. Batch causes the webhook to buffer and send events"+
 			" asynchronously. Known modes are "+strings.Join(pluginwebhook.AllowedModes, ",")+".")
+	fs.IntVar(&o.BatchConfig.BufferSize, "audit-webhook-batch-buffer-size",
+		pluginwebhook.DefaultBatchBufferSize, "The size of the buffer to store events before " +
+			"batching and sending to the webhook. Only used in batch mode.")
+	fs.IntVar(&o.BatchConfig.MaxBatchSize, "audit-webhook-batch-max-size",
+		pluginwebhook.DefaultBatchMaxSize, "The maximum size of a batch sent to the webhook. " +
+			"Only used in batch mode.")
+	fs.DurationVar(&o.BatchConfig.MaxBatchWait, "audit-webhook-batch-max-wait",
+		pluginwebhook.DefaultBatchMaxWait, "The amount of time to wait before force sending the " +
+			"batch that hadn't reached the max size. Only used in batch mode.")
+	fs.Float32Var(&o.BatchConfig.ThrottleQPS, "audit-webhook-batch-throttle-qps",
+		pluginwebhook.DefaultBatchThrottleQPS, "Maximum average number of requests per second. " +
+				"Only used in batch mode.")
+	fs.IntVar(&o.BatchConfig.ThrottleBurst, "audit-webhook-batch-throttle-burst",
+		pluginwebhook.DefaultBatchThrottleBurst, "Maximum number of requests sent at the same" +
```

Comment:
... if ThrottleQPS was not utilized before.

Reply:
Done, thanks